### DISCUSSION TITLE
[Bug/Feature]: Exporting & reviewing functions - Combination of enhancements & possible bugs

### DISCUSSION THREAD

**Discussion by DrSpaldo** [blakeblackshear/frigate#12087] [Report a Bug]
### Describe the problem you are having

I am not sure if these are all intentional design issues or if they could be reworked, probably not considered a "bug" I am guessing. No doubt it is in the eye of the beholder.... 

Please see a few things I have noticed when reviewing footage & trying to capture an incident today. It would be great in the future to be enhanced:

- Exporting multiple cameras; in particular it would make it easier when you are reviewing multiple cameras. Ie. Press review, press motion, select x cameras and then scrub to where you want. This is the place where I would then want to export those cameras for the determined period. 
- Likewise, when in the Review -> Motion page, I cannot go less than 4x speed. I want all the speeds so I can see where things happen, 4 is too fast.
- In the same page, Review -> Motion tab; after selecting x cameras, they don't take up the full space and I cannot make them bigger. So if I select 4 cameras, they are tiny and there is heaps of wasted white/black space.

Hope this makes sense.

### Steps to reproduce

See above

### Version

0.14.0-4e5a6eb

### Frigate config file

```yaml
N/A
```

### Relevant log output

```shell
N/A
```

### Operating system

Proxmox

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

Amcrest

### Any other information that may be helpful

Probably can be changed from bug to feature request?

**Comment by DrSpaldo** [comment]
@NickM-27 I'm not sure why this was closed and is "outdated"? 

They are still missing from the UI as far as I can tell...

**Reply by NickM-27** [reply]
It's not a bug, and each item either has a feature request created for it or is not planned to be implemented
